.github/workflows/actions/changelog/index.js

```diff
@@ -199,7 +199,13 @@ const getChangeLogItems = async (name, owner, sinceDate, to) => {
       hasLabel({ labels }, 'area/grafana/ui') ||
       hasLabel({ labels }, 'area/grafana/toolkit') ||
       hasLabel({ labels }, 'area/grafana/runtime');
-    const author = item.commits.nodes[0].commit.author.user.login;
+    
+    let author = ""
```

Comment:
You may also need to update when it's actually used in `userLink()`

Reply:
I mean this line: https://github.com/grafana/grafana/blob/f8b092aba62115fe28116ddde663ff9909e9bf31/.github/workflows/actions/changelog/index.js#L288

We could do

```typescript
          : `${pullRequestLink(item.number)}${item.author ? ', ' + userLink(item.author) : ''}`
```

instead so the link to author is skipped